src/app/cart/CartSummary.tsx

```diff
@@ -20,14 +20,19 @@ export type WithCheckoutCartSummaryProps = {
 const CartSummary: FunctionComponent<WithCheckoutCartSummaryProps> = ({
     cartUrl,
     ...props
-}) => (
-    withRedeemable(OrderSummary)({
-        ...props,
-        cartUrl,
-        headerLink: (
-            <EditLink url={ cartUrl } />
-        ),
-    })
-);
+}) => {
+    // if there's a sub path name in checkout, assume it's a buy now cart
```

Comment:
If the code isn't self-explanatory, we usually add a comment. Your solution is perfectly comprehensible. This line appears to be a little redundant.